doc/standard_measurement_transcripts.md

```diff
@@ -0,0 +1,68 @@
+# DMTF SPDM Standard Measurement Transcripts
+
+This document describes two standard measurement transcripts for an SPDM Responder that are
+constructed by an SPDM Requester and consumed by a Verifier while all agents are operating in
+production mode. A specification for, or an implementation of, a Verifier or Requester can specify
+and publish support for these transcripts.
+
+## Common Properties
+
+A standard measurement transcript is a byte buffer that consists of the L1/L2 transcript along with
+a signature over the transcript if the Responder supports measurement signing. All
+`GET_MEASUREMENT` requests have the following properties:
+* For SPDM 1.2 and later, `RawBitStreamRequested` is not set.
+* For SPDM 1.3 and later, `NewMeasurementRequested` is not set.
+
+## Single-request Measurement Transcript Definition
+
+The single-request measurement transcript is comprised of a single `GET_MEASUREMENTS` request and a
```

Comment:
```suggestion
The single-request measurement transcript comprises a single `GET_MEASUREMENTS` request and a
```

Reply:
["comprised of"](https://www.merriam-webster.com/dictionary/comprised%20of) is correct.

Reply:
sure, it’s just using more ASCII codepoints :-)